### PR TITLE
Normalize formatting across CMake files

### PR DESCRIPTION
**Pull request by vittorioromeo**
We were using the syntax without a space in most places, so I used that syntax everywhere now..

### CODE REVIEW

**Comment by ChrisThrasher** [comment]
https://github.com/ChrisThrasher/SFML/commit/545e2dc6dad05b27bf1ae1690e27b410fd2b0e91

I wrote a similar commit last week. See if I found any formatting fixes that you missed.

File: cmake/toolchains/iOS.toolchain.cmake [hunk 1]
```diff
@@ -33,156 +33,156 @@
...
-macro (find_host_package)
-	set (CMAKE_FIND_ROOT_PATH_MODE_PROGRAM NEVER)
-	set (CMAKE_FIND_ROOT_PATH_MODE_LIBRARY NEVER)
-	set (CMAKE_FIND_ROOT_PATH_MODE_INCLUDE NEVER)
-	set (IOS FALSE)
+macro(find_host_package)
+	set(CMAKE_FIND_ROOT_PATH_MODE_PROGRAM NEVER)
+	set(CMAKE_FIND_ROOT_PATH_MODE_LIBRARY NEVER)
+	set(CMAKE_FIND_ROOT_PATH_MODE_INCLUDE NEVER)
+	set(IOS FALSE)
```
[the remark this answers is not in the document]
Comment (vittorioromeo) [reply cmake/toolchains/iOS.toolchain.cmake:180]: Sure, will do.

**Review by ChrisThrasher — APPROVED** [review]
I love seeing stuff like this get cleaned up!

**Comment by vittorioromeo** [comment]
SFMLBuildMaster: Build this please.

**Review by eXpl0it3r — APPROVED** [review]

**Pull request merged**